crates/ink/tests/ui/contract/pass/message/message-selector-encoding-sol.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(unexpected_cfgs)]
```

Comment:
Do you remember why this was necessary here?

Reply:
> Do you remember why this was necessary here?
It's because ink! codegen uses `cfg(feature = "ink-as-dependency")`, [`rustc` >= 1.81 checks that `cfg`s match expected names and values](https://blog.rust-lang.org/2024/05/06/check-cfg.html), which are declared with the `--check-cfg` flag. Cargo automatically sets this flag for all features for the target crate ... but in this case, these are ui tests run via `trybuild` with no proper `Cargo.toml` defining the feature. 

That's the short of it anyway 🙂  

> Can you check if it still is?
May be `trybuild` has a way to set the flag, will check.